src/test/java/de/bonndan/nivio/output/map/hex/GroupAreaFactoryTest.java

```diff
@@ -2,12 +2,15 @@
 
 import de.bonndan.nivio.model.*;
 import de.bonndan.nivio.output.map.svg.HexPath;
+import org.apache.commons.collections4.BidiMap;
+import org.apache.commons.collections4.bidimap.DualHashBidiMap;
 import org.apache.commons.collections4.set.UnmodifiableSet;
 import org.junit.jupiter.api.Test;
 
 import java.util.*;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.in;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class GroupAreaFactoryTest {
```

Comment:
Do these tests cover the logic of finding the closest items and the correct occupation of tiles as implemented in the methods `getGroup()`, `addPathsBetweenClosestItems()`, `getClosestItem()` and `getBridges()`?

Reply:
Yes, but it's more like a blackbox test, i.e. we do not test the exact behaviour, but the outcome.

Reply:
Okay, is it hard to add tests for the steps in between? The problem to me is, that it is hard to understand what is going on and also hard to refactor if we only have blackbox tests :-/